packages/devtools_app/lib/src/screens/debugger/breakpoint_manager.dart

```diff
@@ -130,18 +129,18 @@ class BreakpointManager extends Disposer {
   }
 
   void _updateAfterIsolateReload(
-    Event reloadEvent,
+    Event _,
```

Comment:
probably don't need this trailing comma anymore since this should all fit on one line